Cannot get Cmake . to work On Fedora

Note: (Sorry if this issue is listed in a strange way, I'm unfamiliar with github and it's Comment Structure ha ha.)

![Screenshot from 2021-07-12 18-47-06](https://user-images.githubusercontent.com/87342746/125365101-a70da200-e341-11eb-8176-dc5c6e2529bb.png)

**Cmake will output something along the lines of this: 

-- Checking for module 'libevdev'
-- Package 'libevdev', required by 'virtual:world', not found
CMake Error at /usr/share/cmake/Modules/FindPkgConfig.cmake:556 (message):
A required package was not found
Call Stack (most recent call first):
/usr/share/cmake/Modules/FindPkgConfig.cmake:778 (_pkg_check_modules_internal)
CMakeLists.txt:10 (pkg_check_modules)**

Installed the needed package as described in the error, multiple times, restarted my pc, updated repositories, etc 

I honestly have no idea if it's cmake . that is not detecting the package I've downloaded or if I'm doing something wrong (Which is most likely the case because I'm new to fedora.) 

I've been trying to find solutions for this problem for hours now and yet I cannot find anything even remotely related to Fedora when it comes to this software. I've seen different solutions which can all essentially be summed down to "sudo apt-get install libudev-dev" (Which of course is only related to people who uses Ubuntu and other distros of the sort) Which as I stated before I attempted to download the necessary packages and after doing so I still get the same error message. 

For sake of Haste I will be glad to display the Output log file, but within another Comment.

--- a/README.md
+++ b/README.md
@@ -4,7 +4,7 @@ hid-nintendo is currently in review on the linux-input mailing list. The most re
 
 # Installation
 1. clone the repo
-2. Install libevdev-dev if you haven't already (`sudo apt install libevdev-dev`)
+2. Install requirements (`sudo apt install libevdev-dev` or `sudo dnf install libevdev-devel libudev-devel`)
 3. `cmake .`
 4. `sudo make install`
 5. `sudo systemctl enable --now joycond`

